src/cephadm/tests/fixtures.py

```diff
@@ -10,18 +10,22 @@
 from typing import Callable, Dict, List, Optional
 
 
-with mock.patch('builtins.open', create=True):
-    from importlib.machinery import SourceFileLoader
-    cd = SourceFileLoader('cephadm', 'cephadm.py').load_module()
+def import_cephadm():
+    """Import cephadm as a module."""
+    import cephadm as cd
```

Comment:
(nit) `cd` stands for `ceph-daemon` which was the name of this module prior to `cephadm`. Would it be worth replacing all instances of `cd` with something else?

Reply:
Hah. Thank you for explaining the why behind `cd` I thought it was a bit of an overly stretched abbreviation from `_C_epha_D_m` :-)
I'm happy to update it internally and use something like `cephadm` or `_cephadm` in new code, but I don't really want to touch all the many many uses in test_cephadm.py at the moment. Thoughts?

Reply:
`_cephadm` works for me! I prefer consistency and it likely makes sense update all the many instances since we are doing a bit of refactor here, but yeah I also have mixed feelings about touching all the many uses in `tests_cephadm.py`.

Reply:
OK, I'll do a mass conversion in a separate patches and tack them on to this PR.  Once thery're posted we can discuss if it causes too much churn or not... as dropping them from the series wouldn't be difficult.

Reply:
done